cmd/sriov-network-config-daemon/start.go

```diff
@@ -87,6 +88,10 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 
 	snclient := snclientset.NewForConfigOrDie(config)
 	kubeclient := kubernetes.NewForConfigOrDie(config)
+	dynamicclient, err := dynamic.NewForConfig(config)
```

Comment:
Can you use NewForConfigOrDie (as per the others)?
Also, can the name be `dynamicClient`?

Reply:
Done